src/clj/game/cards/events.clj

```diff
@@ -501,6 +501,20 @@
                                              (gain-credits :runner (five-or-all corp)))})}
                       card))}
 
+   "Divide and Conquer"
+   {:req (req archives-runnable)
+    :makes-run true
+    :async true
+    :effect
+    (effect (run :archives
+                 {:end-run
+                  {:async true
+                   :effect
+                   (req (wait-for (do-access state side [:rd] {:no-root true})
```

Comment:
Do we have confirmation that the R&D access benefits from additional accesses?

Reply:
[Yep.](http://7tr.in/ANR/divide-and-conquer.png) Gonna be in the forthcoming UFAQ.